client/src/locales/en-US.json

```
@@ -306,6 +306,15 @@
         "moveHere": "Move here",
         "cancel": "Cancel"
     },
+    "LongPathsSupportModal": {
+        "title": "Support for long paths",
+        "closeButton": "Close",
+        "noReminder": "Don't remind me",
```

To be consistent with french version
```suggestion
        "noReminder": "Don't remind me again",
```